### ISSUE TITLE
Variation of chams that only work through walls

### ISSUE THREAD

**Issue by P0gDog** [iiDk-the-actual/iis.Stupid.Menu#123]
**Is your feature request related to a problem? Please describe.**
I am always frustrated when I am using chams and it gets hard to see people when you are up close

**Describe the solution you'd like**

A version of chams that only shows people through walls, but shows regular vision when there is no obstruction 
 
**Describe alternatives you've considered**

An outline of players instead so you can still see the primary body

**Comment by iiDk-the-actual** [comment]
I'm not sure how I would do this without a custom shader, which you can't really import to URP properly

**Comment by P0gDog** [comment]
Is there any other solution you can think of?

**Comment by P0gDog** [comment]
I just got an idea. Maybe something similar to esp in fps games where there torso and arms have lines in them. This can provide more information and more visibility.

**Comment by iiDk-the-actual** [comment]
Wireframe?

**Comment by P0gDog** [comment]
Not quite. Similar? But there is only three lines. One through each arm and one through the torso.

**Comment by monkeybot153** [comment]
> Not quite. Similar? But there is only three lines. One through each arm and one through the torso.

I think you are talking about bone esp, its already in the menu.

**Comment by P0gDog** [comment]
Dang, can’t believe I didn’t notice. Thanks.
